apache/superset
# fix: add feature flag to header bar

### SUMMARY
Removes the add report icon from the header bar when feature flag is not enabled. Also prevents fetching the report list without proper access to the endpoint. 
![USA_Births_Names](https://user-images.githubusercontent.com/5186919/127897983-af073cfb-1e11-4b8a-bd47-39aabe0b8bd1.png)

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS
With feature flag off you should not be able to create a report from a chart or dashboard. There should be no buttons or any sort of indication that you can create a report. 

### ADDITIONAL INFORMATION
